packages/core/src/scd-queries/ied/queries.ied.ts

```diff
@@ -0,0 +1,93 @@
+import { SCDQueries } from '@/scd-queries/scd-queries'
+// CONSTANTS
+import { SCD_ELEMENTS } from '@/constants'
+// TYPES
+import type {
+	CommonOptions,
+	IEDElement,
+	IEDElementStandardAttributes,
+	ReportControlElement,
+	ClientLNElement,
+	InputsElement,
+	ExtRefStandardAttributes,
+	ExtRefElement,
+	GSEControlElement
+} from '@/scd-queries'
+
+export class IEDQueries extends SCDQueries {
+	//====== PUBLIC METHODS
+
+	public static SelectorIED = SCD_ELEMENTS.ied.element.tag
+	public static AttributeListIED: Array<keyof IEDElementStandardAttributes> =
+		['name']
+	public searchIEDs(options?: CommonOptions): IEDElement[] {
+		return this.searchElement<IEDElement>(
+			IEDQueries.SelectorIED,
+			IEDQueries.AttributeListIED,
+			options
+		)
+	}
+
+	public static SelectorReportControl = SCD_ELEMENTS.reportControl.element.tag
+	public searchReportControls(
+		options?: CommonOptions
+	): ReportControlElement[] {
+		return this.searchElement<ReportControlElement>(
+			IEDQueries.SelectorReportControl,
+			['rptID', 'name', 'datSet'],
+			options
+		)
+	}
+
+	public static SelectorClientLN = SCD_ELEMENTS.clientLN.element.tag
+	public searchClientLNs(options?: CommonOptions): ClientLNElement[] {
+		return this.searchElement<ClientLNElement>(
+			IEDQueries.SelectorClientLN,
+			['iedName'],
+			options
+		)
+	}
+
+	public static SelectorInput = SCD_ELEMENTS.inputs.element.tag
+	public searchInputs(options?: CommonOptions): InputsElement[] {
+		return this.searchElement<InputsElement>(
+			IEDQueries.SelectorInput,
+			[],
+			options
+		)
+	}
+
+	public static SelectorExtRef = SCD_ELEMENTS.extRef.element.tag
+	public static AttributeListExtRef: Array<keyof ExtRefStandardAttributes> = [
+		'iedName',
+		'serviceType',
+		// "ldInst",
+		// "lnClass",
+		// "lnInst",
```

Comment:
Is there a reason for leaving them commented out?

Reply:
This comes from the legacy code, I forgot to clean them ;)